CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to this project will be documented in this file.
 - Added a `filename` attribute to UVCal, UVBeam and UVFlag objects.
 - Support for reading MWAX/birli uvfits files.
 - Flexible spectral windows to `read_mwa_corr_fits`.
+- Reading writing of scan numbers for MS files as `UVData.scan_number_array`.
+- Grouping of contiguous integrations for a phase center into "scan numbers" in `UVData._set_scan_numbers`.
+This grouping defines `UVData.scan_number_array` when not originally present in the data (e.g. reading in
+a non-MS file) and is used when writing to an MS file.
```

Comment:
These should go to the top of this section. The changelog is in reverse chronological order.

Reply:
Fixed! Thank you!